mobile/mobile WatchKit Extension/ExtensionDelegate.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  ExtensionDelegate.swift
+//  mobile WatchKit Extension
+//
+//  Created by Collin Prather on 3/24/20.
+//  Copyright © 2020 Collin Prather. All rights reserved.
+//
+
+import WatchKit
```

Comment:
I wonder if there is an equivalent way to do this using SwiftUI instead of WatchKit.

Reply:
My thoughts exactly. As I understand, SwiftUI was kinda a replacement for UIKit, but I'm not sure if it directly replaced things like WatchKit. Definitely interested if we find a better way though!